packages/frontend/@n8n/design-system/src/components/N8nPopoverReka/N8nPopoverReka.vue

```diff
@@ -48,22 +59,28 @@ const emit = defineEmits<Emits>();
 		<PopoverTrigger :as-child="true">
 			<slot name="trigger"></slot>
 		</PopoverTrigger>
+		<PopoverAnchor v-if="reference" :reference="reference" />
 		<PopoverPortal>
-			<PopoverContent side="bottom" :align="align" :side-offset="5" :class="$style.popoverContent">
+			<PopoverContent
+				:side="side"
+				:side-flip="sideFlip"
+				:align="align"
+				:side-offset="sideOffset"
+				:class="[$style.popoverContent, contentClass]"
+				:style="{ width }"
+			>
 				<N8nScrollArea
 					v-if="enableScrolling"
 					:max-height="props.maxHeight"
 					:type="scrollType"
 					:enable-vertical-scroll="true"
 					:enable-horizontal-scroll="false"
 				>
-					<div :style="{ width }">
```

Comment:
Moving `width` to PopoverContent because its border increases the width of the popover by 2px which is not negligible when the equal width with the anchor is important